### DISCUSSION TITLE
Include previous image tag in imageupdateautomation notifications

### DISCUSSION THREAD

**Discussion by tun0** [fluxcd/flux2#3027] [Proposals] [answered]
The notifications we currently have with Flux v1 (with help of fluxcloud) mention both old and new image tags.

With Flux v2, only the new image tag is mentioned.

It would be very nice if the previous tag would be made available as well in the notification object.

**Comment by kingdonb** [comment, marked as answer]
This could be possible! We just have been talking about this at Bug Scrub, based on your feedback here. Let's create an issue for this feature request in the Image Automation repository. 👍 (https://github.com/fluxcd/image-automation-controller/issues/428)
